index.html

```diff
@@ -2,49 +2,55 @@
 
 <head>
     <title>Nikos Village Website lol</title>
-    <link rel="styleSheet" href="style.css"/>
+    <link href="style.css" rel="styleSheet"/>
 </head>
 
 <body>
-    
```

Comment:
My bad on messing up these spaces, I will admit it looks far better now that its formatted.

Reply:
Not a mistake, just pointing out how it can be done :) 

All IDEs have a way to format code. Look up how to format code with VScode and make it a habit of doing before you commit to a pr